### PR TITLE
CMQ-1306: Add a change log to the specification

### PR DESCRIPTION
**Pull request by czeczotka**

### CODE REVIEW

File: source/index.html.md.erb [hunk 1]
```diff
@@ -8,11 +8,8 @@ description: You need to send header data to HMRC when you use our APIs. Check t
 
 # Send fraud prevention data
 
-Version 2.9 issued 14 July 2020
-
-<div class="govuk-inset-text">
-We have redesigned this document to help you find the information you need. There are no changes to the data you need to send. 
-</div>
+Version 3.0 issued 16 November 2020 <br/>
+<a href="/getting-it-right/#change-log" onclick="ga('send', 'event', 'On Page Navigation', 'Send fraud prevention data', 'Change log');">Check what has changed</a>
```
[the remark this answers is not in the document]
Comment (edjannoo) [reply source/index.html.md.erb:12]: I think that might be the only way to do it

File: source/getting-it-right/index.html.md [hunk 2]
```diff
@@ -74,3 +74,14 @@ If you use or plan to use third-party software and libraries, make sure you can
...
+## Change log
+
+### Version 3.0
+
+16 November 2020
+
+What changed:
+
+* Gov-Vendor-Product-Name required for all connection methods
+* Gov-Client-Local-IPs-Timestamp required with Gov-Client-Local-IPs
```
Comment (edjannoo) [inline source/getting-it-right/index.html.md:87]: We also have Gov-Vendor-Public-IP-Timestamp as a new header

File: source/index.html.md.erb (hunk 1 above)
[the remark this answers is not in the document]
Comment (czeczotka) [reply source/index.html.md.erb:12]: 🤷‍♂️

File: source/getting-it-right/index.html.md (hunk 2 above)
Comment (czeczotka) [reply source/getting-it-right/index.html.md:88]: Thanks Ed! I knew I missed something. I added it now but the idea was to revisit it later anyway.

**Pull request merged**